metriks/metriks.go

```diff
@@ -0,0 +1,124 @@
+package metriks
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/armon/go-metrics"
+	"github.com/armon/go-metrics/datadog"
+)
+
+const timerGranularity = time.Millisecond
```

Comment:
What about making this configurable? Don't see a clear value of having this const here.